content/id/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Mengatur Service Account untuk Pod
+content_type: task
+weight: 90
+---
+
+<!-- overview -->
+Akun servis (_service account_) menyediakan identitas untuk proses yang sedang berjalan dalam sebuah Pod.
+
+{{< note >}}
+Dokumen ini digunakan sebagai pengenalan untuk pengguna terhadap _Service Account_ dan menjelaskan bagaimana perilaku _service account_ dalam konfigurasi kluster seperti yang direkomendasikan Kubernetes. Pengubahan perilaku yang bisa saja dilakukan administrator kluster terhadap kluster tidak menjadi bagian pembahasan dokumentasi ini.
+{{< /note >}}
+
+Ketika kamu mengakses kluster (contohnya menggunakan `kubectl`), kamu terautentikasi oleh apiserver sebagai sebuah User Account (untuk sekarang umumnya sebagai `admin`, kecuali jika administrator klustermu telah melakukan pengubahan). Berbagai proses yang ada di dalam kontainer dalam pod juga dapat mengontak apiserver. Ketika itu terjadi, mereka akan diautentikasi sebagai sebuah Service Account (contohnya sebagai `default`).
+
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+
+## Menggunakan Default Service Account untuk Mengakses API server.
```

Comment:
nit: no need for period on header
```suggestion
## Menggunakan Default ServiceAccount untuk Mengakses API server
```

Reply:
I use period since the original English page use one. I just want to confirm that let's just not use period at the end of a header?

Reply:
yes we can remove, sometimes there are errors in original english pages as well